release_notes.md

```diff
@@ -1,59 +1,10 @@
 # What's New
 
-## Support for FortiAI GenAI Assistant
+## Support for Voice Recognition in AI Assistant
```

Comment:
New release note as that now we are not deleting the thread from OpenAI, we are only clearing the conversation on clicking the 'Clear Conversation' from bot

Reply:
same thread? not new thread? @cs-ankit-gupta